src/main/java/net/mcreator/ui/laf/MCreatorTheme.java

```diff
@@ -72,19 +76,25 @@ public Color getMainTint() {
 		return MAIN_TINT;
 	}
 
-	@Nonnull protected abstract Color getBlackAccent();
-
-	@Nonnull protected abstract Color getDarkAccent();
-
-	@Nonnull protected abstract Color getLightAccent();
+	private @Nonnull Color getBlackAccent() {

```

Comment:
As these are no longer used for overrides you can drop these methods and inline them inside this class.